src/oop/app/src/ScrapperFactory.php

```diff
@@ -19,11 +21,11 @@ public function create(string $domain): Scrapper
     {
         switch ($domain) {
             case 'filmix':
-                return new Scrapper(new CurlStrategy(), new FilmixParserStrategy());
+                return new Scrapper(new CurlStrategy(), new FilmixParserStrategy(new Movie()));
             case 'kinoukr':
-                return new Scrapper(new GuzzleAdapter(), new KinoukrDomCrawlerParserAdapter());
+                return new Scrapper(new GuzzleAdapter(), new KinoukrDomCrawlerParserAdapter(new Crawler(), new Movie()));
```

Comment:
You did not pass Client to GuzzleAdapter

Reply:
> You did not pass Client to GuzzleAdapter

Sorry, forgot to pass the content of the local file.